tests/test_core_utils.py

```diff
@@ -24,3 +26,13 @@ def test_accessing_hydra_config(hydra_restore_singletons: Any) -> Any:
         del cfg["hydra"]
     assert cfg.job_name == "UNKNOWN_NAME"
     assert cfg.config_name == "accessing_hydra_config"
+
+
+def test_py_version_resolver(hydra_restore_singletons: Any, monkeypatch: Any) -> Any:
+    Version = namedtuple("Version", "major minor micro")
```

Comment:
cam you drop the namedtuple here?
I don't think it's worth depending on it for such a tiny thing.

Reply:
:( I'm not sure how else to mock this 😰

Reply:
This:
```
version = (3,7,2)
```
Does not work?

Reply:
> This:
> 
> ```
> version = (3,7,2)
> ```
> 
> Does not work?

didn't work before because I was calling ```sys.version_info.major```, changed the code to ```sys.version_info[0]``` now and removed the dependency.

Reply:
Yeah, it's a tuple although it has bigger aspirations :) :
https://docs.python.org/3.1/library/sys.html#sys.version_info

```python
In [10]: sys.version_info == (3,8,2,"final", 0)
Out[10]: True
```